content/docs/2.5/scalers/newrelic.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "New Relic"
+availability = "2.5+"
+maintainer = "Community"
+description = "Scale applications based on NewRelic NRQL"
+layout = "scaler"
+go_file = "newrelic_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `New Relic` trigger that scales based on a New Relic metric.
+
+```yaml
+triggers:
+  - type: new-relic
+    metadata:
+      # Required: Account - Subaccount to run the query on
+      Account: 1234567
+      # Required: QueryKey - Api key to connect to New Relic
+      QueryKey: "NRAK-xxxxxxxxxxxxxxxxxxxxxxxxxxx"
```

Comment:
This should be imho stored in a Secret - thus TriggerAuthentication

Reply:
Added a request on the feature PR as well

Reply:
If I'm reading https://github.com/kedacore/keda/blob/5c163a617fc8c68a715a64a322372cf1acfd57a3/pkg/scalers/scaler.go#L90 (GetFromAuthOrMeta) correctly and is used in https://github.com/kedacore/keda/pull/2286/files#diff-747ec231fcf7a74b8b06f6ab11c31078aaf0ae04f333d788562179a3733ebfebR86. This should be able to be pulled currently from the Auth based setup. So I've added/updated based on that understanding.